content/sublease.md

```diff
@@ -0,0 +1,52 @@
+# SUBLEASE AGREEMENT
+
+This Sublease Agreement (the "Agreement") is made on [DATE] between G. Ochoa LLC ("Sublessor") and [SUBLESSEE NAME] ("Sublessee").
+
+WHEREAS, Sublessor leases certain premises located at 544 Park Ave, Suite 332, New York, NY 12205 (the "Premises") and wishes to sublease a portion of the Premises to Sublessee.
+
+NOW, THEREFORE, in consideration of the mutual covenants herein, the parties agree as follows:
+
+1. SUBLEASED SPACE: Sublessor agrees to sublease to Sublessee a portion of the Premises (the "Subleased Space") for activities.
```

Comment:
"for ... activities"? should this either reference the allowed activities below, or describe them generally? perhaps "for workshop activities"